docs/use_cases/index.rst

```diff
@@ -9,7 +9,7 @@ Physical Oceanography
 .. toctree::
    :maxdepth: 1
 
-   ocean_cm2.6_eddy_statistics
+   sea-surface-height
```

Comment:
do you want to remove the `ocean_cm2.6_eddy_statistics` notebook since  in this PR too?

Reply:
Yes. That notebook is out of date. I will be adding an updated one in a future PR.